modules/components/pages/outputs/quest_db.adoc

```diff
@@ -0,0 +1,423 @@
+= questdb
+:type: output
+:status: experimental
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Pushes messages to a QuestDB table.
+
```

Comment:
MIssing the conditionalized version in which it was introduced?

Reply:
Added along with single sourcing. I've left the beta tag outside of the single sourcing so that this is checked before the change goes live in Cloud.